File: 2sem/lab8/lab8_sharp/Node.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace lab8_sharp
{
    class Node
    {
        public double Value { get; set; }  // property that contains some float value
        public Node Next { get; set; }  // property that contains next element

        public Node(double num)  // constructor
        {
            Value = num;
            Next = null;
        }
    }
}

File: 2sem/lab8/lab8/Node.cpp
#include "Node.h"

Node::Node(float num) {  // constructor
	value = num;
	next = nullptr;
}

void Node::SetNext(Node* next) {  // next element setter
	this->next = next;
}

void Node::SetValue(float num) {  // value setter
	value = num;
}

Node* Node::GetNext() {  // next element getter
	return next;
}

float Node::GetValue() {  // value getter
	return value;
}